### PR TITLE
fix: Apply design changes from figma

### PR DESCRIPTION
**Pull request by samuveth**
This is mainly fixing the layout for space delegates page

Other changes:
- Add divider in the sidebar
- Fix delegate card layout
- If not following show Join button as primary, otherwise show delegate/new proposal as primary button

### CODE REVIEW

**Review by wa0x6e — COMMENTED** [review]
Also on medium screen, the confirm box is missing spacing

Screenshot 2023-08-08 at 21 55 08

File: src/components/ButtonFollow.vue [hunk 1]
```diff
@@ -32,6 +32,7 @@ const canFollow = computed(() =>
         'flex items-center justify-center hover:!border-red hover:!bg-red hover:!bg-opacity-5 hover:!text-red':
           isFollowing
       }"
+      :primary="!isFollowing"
```
Comment (wa0x6e) [inline src/components/ButtonFollow.vue:35]: Is it a good idea UX wise to have the same button with the same meaning have 2 differents UI ?

Comment (samuveth) [reply src/components/ButtonFollow.vue:35]: I don't see the issue. Also this idea comes from designers

**Comment by samuveth** [comment]
> Also missing spacing after the loading placeholder

This page is work in progress and not been changes in this PR

**Pull request merged**